class-udm-updater.php

```diff
@@ -43,6 +43,10 @@ class Updraft_Manager_Updater_1_8 {
 	 */
 	public function __construct($mothership, $muid, $relative_plugin_file, $options = array()) {
 
+		global $wp_version;
```

Comment:
@andtown `include()` brings anything defined in the included file into scope. Therefore, `global $wp_version;` is no longer needed.